src/buy/buyers-guide.md

```diff
@@ -90,8 +90,8 @@ why not manufacture a [deck PCB](https://github.com/bitcraze/crazyflie2-exp-temp
 {% row_links %}
 **What to buy:**
 
-* [Breakout deck](https://store.bitcraze.io/products/breakout-deck)
```

Comment:
Let's perhaps leave the direct to store guides links for the buyers guide. It is quite seperated anyway and it is meant to help people to check what their are looking for.